pages/linux/serialver.md

```diff
@@ -0,0 +1,21 @@
+# serialver
+
+> Returns the serialVersionUID of classes.
+> It does not set a security manager by default.
+> More information: <https://manned.org/serialver.1>.
+
+- Display serialVersionUID of a class:
+
+`serialver {{classnames}}`
+
+- Display the user interface, enter full class name and press show to view the serial version:
+
+`serialver -show {{classnames}}`
```

Comment:
I checked the command and this `-show` flag is not available in the latest version. Just remote it.

Reply:
done